src/printer/index.js

```diff
@@ -189,10 +150,29 @@ function printIf(path, print) {
     indent(concat([hardline, printBody(path, print)]))
   ];
 
-  const orElse = printOrElse(path);
+  while (n.orelse && n.orelse.length === 1 && n.orelse[0].ast_type === "If") {
+    n = n.orelse[0];
```

Comment:
maybe just something like:

```js
parts.push(concat(path.map(printElif, 'orelse')));
```

and then have `printElif` deal with this `if` clause, or even like a `printIfBodyAndOrElse`

Reply:
I'm not sure I'm understanding what you mean here